plugin/web/components/sidebar/Layout/MainLayout.tsx

```diff
@@ -0,0 +1,66 @@
+// import " antd/dist/antd.less";
+import { Layout, MenuProps } from "antd";
+import { MenuInfo } from "rc-menu/lib/interface";
+import { memo, useMemo, useState } from "react";
+
+import { ReactComponent as DataBase } from "../../common/Icon/Icons/dataBase.svg";
+import { ReactComponent as Info } from "../../common/Icon/Icons/info.svg";
+import { ReactComponent as Share } from "../../common/Icon/Icons/share.svg";
+import { ReactComponent as Sliders } from "../../common/Icon/Icons/sliders.svg";
+import { ReactComponent as Template } from "../../common/Icon/Icons/template.svg";
+
+import LayoutContent from "./LayoutContent";
+import LayoutFooter from "./LayoutFooter";
+import LayoutHeader from "./LayoutHeader";
+
+const items: MenuProps["items"] = [
+  {
+    key: "mapData",
+    icon: <DataBase />,
+  },
+  {
+    key: "mapSetting",
+    icon: <Sliders />,
+  },
+  {
+    key: "shareNprint",
+    icon: <Share />,
+  },
+  {
+    key: "about",
+    icon: <Info />,
+  },
+  {
+    key: "template",
+    icon: <Template />,
+  },
+];
+export type Props = {
+  className?: string;
+  isInsideEditor: boolean;
+};
+
+const WdLayout: React.FC<Props> = ({ className, isInsideEditor }) => {
+  const [current, setCurrent] = useState("mapData");
```

Comment:
Please type this. Currently it'll just be typed as string.

Reply:
done

Reply:
https://github.com/eukarya-inc/reearth-plateauview/pull/29/commits/326aa3062b26c0e076d4cc56cf8ad8237c0cf1e2